test/RuntimeAgent.js

```diff
@@ -171,21 +171,16 @@ describe('RuntimeAgent', function() {
             return prop.name == '__proto__';
           });
           expect(proto.length == 1, 'proto exist and unique').to.equal(true);
-          // expect(proto[0], '__proto__ has valid structure').to.deep.equal({
-          //     name: '__proto__',
-          //     value: {
-          //       type: 'object',
-          //       subtype: undefined,
-          //       objectId: '17',
-          //       className: 'Object',
-          //       description: 'InspectedClass'
-          //     },
-          //     writable: true,
-          //     configurable: true,
-          //     enumerable: false,
-          //     isOwn: true
-          //   });
-
+          var pro = proto[0];
+          expect(pro.name).to.equal('__proto__');
```

Comment:
ditto - it may be enough to replace `.deep.equal` with `.deep.include` or `.deep.contains` + remove `objectId` from the expected value.

you may need to upgrade `chai` to the latest version first.